src/target/aby/trans.rs

```diff
@@ -246,24 +273,38 @@ impl ToABY {
         let share = self.get_share_name(&t);
         let s = self.term_to_share_cnt.get(&t).unwrap();
         match &t.op {
-            Op::Var(_, Sort::BitVector(_)) => {
+            Op::Var(name, Sort::BitVector(_)) => {
+                if !self.inputs.contains(&t) && self.md.input_vis.contains_key(name) {
+                    let term_name = ToABY::get_var_name(&t);
```

Comment:
Why is it that for the other helper functions, you use self."function_name", but for get_var_name(), you use ToABY::get_var_name()?

Is it because get_var_name() does not need to take &self as an argument?

Reply:
`self.get_share_name()` needs to reference `self.term_to_share_cnt`, which requires `self`.

`get_var_name` is getting the string from a `Term`, which doesn't require `self`. 

You can look through the function definitions for a better understanding!